src/components/CommentsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+function CommentsList() {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch(
+      `https://kirils-ncnews.herokuapp.com/api/articles/${article_id}/comments`
+    )
+      .then((res) => {
+        return res.json();
+      })
+      .then((response) => {
+        setComments(response.commentsList);
+        setIsLoading(false);
+      });
+  }, [article_id]);
+  if (isLoading) return <p>Loading...</p>;
+
+  return (
+    <section>
+      <h3>CommentsList:</h3>
+      {comments.map(({ comment_id, body, author, votes, created_at }) => {

```

Comment:
I think it might be more semantic to have the list of comments be inside a list element